webapp/src/views/Console/DomainDetailsDialog.vue

```diff
@@ -24,7 +24,8 @@
       </v-alert>
       <v-card-text>
         <p>
-          To properly secure your domain with DNSSEC, please forward the following information to your domain registrar:
+          Please forward the following information to the administrator/registry of
+          <code>.{{name.split('.').slice(1).join('.')}}</code>:
```

Comment:
I think it would be better to remove the left-most dot.

Reply:
With the text as suggested below, just {{name}} is fine.